Automattic/jetpack
# Remove redux store

## Proposed changes:

* Fully remove redux store as all of the used data has been migrated out of it
* Remove welcome banner hook leftover from the welcome banner data migration

### Other information:


## Jetpack product discussion

P2: p1HpG7-rcF-p2

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

1. Checkout this branch via the Jetpack Beta plugin or your local dev environment
2. Make sure My Jetpack loads with no errors or console errors and works correctly still. We aren't removing any functionality here so just have to make sure there's no broken imports or anything
